doc/en/components/general-changelog-dv-blazor.md

```diff
@@ -10,6 +10,81 @@ namespace: Infragistics.Controls.Charts
+- `Combo`, `DatePicker`, `Dialog`, `Dropdown`,  `ExpansionPanel`, `NavDrawer`, `Toast`, `Snackbar`, **IgbSelectComponent**
+  - Toggle methods `Show`, `Hide`, `Toggle` methods return **true** now on success. Otherwise **false**.
+
+**BREAKING CHANGES**:
+
+- Renamed old `DatePicker` to **IgbXDatePicker**.
+- Removed `Form` component. Use native form instead.
+- Removed `size` property in favor of the `--ig-size` CSS custom property for the following components:
+  - `Avatar`, `Button`,`IconButton`, `Calendar`, `Chip`, `Dropdown`, `Icon`, `Input`, `List`, `Rating`, `Snackbar`, `Tabs`, `Tree`
+- `Badge`, `Chip`, `LinearProgress`, `CircularProgress`
+  - Renamed `Variant` property type to `StyleVariant`.
+- `Calendar`
+  - Renamed `WeekStart` property type to `WeekDays`.
+- `Checkbox`, `Switch`
+  - Changed `Change` event argument type from `ComponentBoolValueChangedEventArgs` to `CheckboxChangeEventArgs`.
+- `Combo`
+  - The `IgbCombo` is now of generic type and its type needs to be specified either by setting `T` or other methods. That way binding to `Value` allows for collection as well.
+  - Removed `PositionStrategy`, `Flip`, `SameWidth` properties.
+- `SelectComponent`
+  - Removed `PositionStrategy`, `Flip`, `SameWidth` properties.
+- `DateTimeInput`
+  - Removed `MaxValue` and `MinValue` properties. Used `Max` and `Min` instead.
+- `Dialog`
+  - Renamed `CloseOnEscape` property to `KeepOpenOnEscape`.
+- `Dropdown`
+  - Removed `PositionStrategy` property.
+- `Input`
+  - Removed `Maxlength` and `Minlength` properties. Use the `Max` and `Min` properties instead.
+  - Renamed `Readonly` and `Inputmode` properties to `ReadOnly` and `InputMode`.
+  - Changed `InputMode` type also to `string`.
+- `Radio`
+  - Changed `Change` event argument type from `ComponentBoolValueChangedEventArgs` to `RadioChangeEventArgs`.
+- `RangeSlider`
+  - Removed `AriaThumbLower` and `AriaThumbUpper` properties. Use `ThumbLabelLower` and `ThumbLabelUpper` instead.
+- `Rating`
+  - Renamed `Readonly` property to `ReadOnly`.
+
+### {PackageGrids}
+
+- `PivotGrid`
+  - Added `Sortable` property for a `PivotDimension`.
+  - Added horizontal layout. Can be enabled inside the new `PivotUI` property as `RowLayout` `Horizontal`.
+  - Added row dimension summaries for horizontal layout only. Can be enabled for each `PivotDimension` by setting `HorizontalSummary` to **true**.
+  - Added `HorizontalSummariesPosition` property to the `PivotUI`, configuring horizontal summaries position.
+  - Added row headers for the row dimensions. Can be enabled inside the new `PivotUI` property as `ShowHeaders` **true**.
+  - Keyboard navigation now can move in to row headers back and forth from any row dimension headers or column headers.
+  - Added keyboard interactions for row dimension collapse using `Alt + Arrows` and row headers sorting using `Ctrl + Arrow Up/Down`.
+
+**BREAKING CHANGES**:
+- `All Grids`, `RowIsland`
+  - Removed `DisplayDensity` deprecated property.
```

Comment:
I think it would be nice to have a link to a topic explaining sizing in the world without displayDensity. I know we have it for Angular and if we don't for xplatform it should be added to our backlog.